cherrypy/_helper.py

```diff
@@ -308,3 +308,34 @@ def normalize_path(path):
         newpath = '/' + newpath
 
     return newpath
+
+
+####
+# Inlined from jaraco.classes 1.4.3
```

Comment:
@jaraco I think, better place for such code vendoring would be some `lib/vendor` folder, which is a git submodule (or a subtree, but I don't have real experience with latter), which is easier to keep in sync. Also it would be easy to embed them into distribution.

Reply:
Well, I don't want to vendor the whole package. I also see this as a very temporary pattern until #1673 lands, so probably not worth too much extra effort. But if you'd like to take a stab at it, you're welcome to do so.

Reply:
it's not that hard to implement, I'll probably find time for this